tests/CppUTestExt/MockActualCallTest.cpp

```diff
@@ -133,3 +142,56 @@ TEST(MockCheckedActualCall, MockIgnoredActualCallWorksAsItShould)
     CHECK_FALSE(actual.hasReturnValue());
     CHECK(actual.returnValue().equals(MockNamedValue("")));
 }
+
+TEST(MockCheckedActualCall, remainderOfMockActualCallTraceWorksAsItShould)
+{
```

Comment:
Break this one up in smaller tests? It started out like MockIgnoredCall, except that there is a lot more interesting stuff to check here.